src/values/NativeFunctionValue.js

```diff
@@ -83,6 +83,10 @@ export default class NativeFunctionValue extends FunctionValue {
     }
   }
 
+  hasDefaultLength(): boolean {
+    return true;
+  }
+
```

Comment:
Since the parent class FunctionValue defines hasDefaultLength() as abstract method(throws exception), I will just remove hasDefaultLength() from NativeFunctionValue. Anyone calls hasDefaultLength() on NativeFunctionValue will just throw by default.